domFoundations/events,storage,attributes/initialPageRenderer.js

```diff
@@ -0,0 +1,21 @@
+import { loadData } from "./storage.js";
+
+export const backgr = document.querySelector('body');
+export const dateField = document.querySelector('.dateOutput');
+export const btnOnOff = document.querySelector('.btnOnOff');
+export const dateOnOff = document.querySelector('.dateOnOff');
+export const dateText = document.querySelector('.date');
+export const timeText = document.querySelector('.time');
+export const textOutputdocument= document.querySelector('.dateOutput');
```

Comment:
It's better to have a apex `Element` in var names related to HTML elements

Reply:
Done